nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -76,6 +76,12 @@ let
     else
       max (config.boot.loader.timeout * 10) 1;
 
+  # Timeout in grub is in seconds.
+  # 0 boots the default target immediately, -1 waits forever.
+  grubEfiTimeout = if config.boot.loader.timeout == null then
+      0
```

Comment:
you meant `-1` here I think

Reply:
Here -- where? `-1` instead of `0`?

I think the `0` default for syslinux means "no timeout, boot immediately" right? <https://wiki.syslinux.org/wiki/index.php?title=SYSLINUX#TIMEOUT_timeout>

Which I think is the same as `0` for grub: <https://www.gnu.org/software/grub/manual/grub/html_node/timeout.html>

Reply:
Yes, therefore, it's wrong, see my comment above, the description of `boot.loader.timeout` explicitly say that `null` means **infinite** wait, not **zero wait**.

If syslinux does it wrong, it's a problem, albeit not this PR's one :P and we should fix it / open an issue to track it.

Reply:
https://github.com/NixOS/nixpkgs/issues/207289